tests/app/conftest.py

```diff
@@ -421,29 +430,34 @@ def sample_notification_with_job(
 
 
 @pytest.fixture(scope='function')
-def sample_notification(notify_db,
-                        notify_db_session,
-                        service=None,
-                        template=None,
-                        job=None,
-                        job_row_number=None,
-                        to_field=None,
-                        status='created',
-                        reference=None,
-                        created_at=None,
-                        sent_at=None,
-                        billable_units=1,
-                        personalisation=None,
-                        api_key_id=None,
-                        key_type=KEY_TYPE_NORMAL,
-                        sent_by=None,
-                        client_reference=None):
+def sample_notification(
+    notify_db,
+    notify_db_session,
+    service=None,
+    template=None,
+    job=None,
+    job_row_number=None,
+    to_field=None,
+    status='created',
+    reference=None,
+    created_at=None,
+    sent_at=None,
+    billable_units=1,
+    personalisation=None,
+    api_key_id=None,
+    key_type=KEY_TYPE_NORMAL,
+    sent_by=None,
+    client_reference=None,
+    updated_at=None
```

Comment:
rather than change this function, can you use `create_notification` in `api/tests/db.py`? that way we don't need to faff about with notify_db/notify_db_session in tests